.release/ci.hcl

```diff
@@ -109,6 +109,7 @@ event "promote-staging" {
     organization = "hashicorp"
     repository = "crt-workflows-common"
     workflow = "promote-staging"
+    config = "release-metadata.hcl"
```

Comment:
Any reason not to do this for production as well?

Reply:
Great question! @sarahethompson will be able to answer this. I will ping her.

Reply:
👋 - The release metadata is only needed for the initial publish (to staging). During the `promote-production` workflow we do a (new) `hc-releases promote` which promotes the files and metadata from staging to prod.